server/src/main/scala/passengersplits/polling/ManifestFilePolling.scala

```diff
@@ -0,0 +1,289 @@
+        val manifestFuture: Future[Seq[FileSourceAndOptManifest]] = zipFuture
+          .map(manifests => {
+            val jsonFilenames = manifests.map(_.filename)
+            logInfo(s"processing manifests from zip. Length: ${manifests.length}, Content: ${jsonFilenames}")
+            manifestsToAdvPaxReporter(logInfo, flightPassengerReporter, manifests)
+          })
+
+        val successfulJsonManifests = manifestFuture.map { manifestOpts =>
+          val (manifests, errors) = manifestOpts partition {
+            case (_, _, Success(_)) => true
+            case _ => false
+          }
+          
+          errors.foreach {
+            case (_, jsonFile, Failure(f)) => logWarn(s"""Failed to parse voyage passenger info: "$jsonFile", error: $f""")
+          }
+
+          logInfo(s"processed manifests: Length ${manifestOpts.length}  ${manifestOpts.headOption.map(_._2)}")
+          manifests.map { case (_, _, Success(vpi)) => vpi }
+        }
+
+
+        val promiseZipDone = Promise[ZipFileName]()
+        val monitor = ManifestFilePolling.zipCompletionMonitor(actorSystem, ManifestFilePolling.props(promiseZipDone))
+        log.debug(s"tickId: $tickId zipCompletionMonitor is $monitor")
+        val subscriberFlightActor = ManifestFilePolling.subscriberFlightActor(flightPassengerReporter,
+          VoyageManifestZipFileComplete(zipFileName, monitor))
+
+        val eventualZipCompletion = promiseZipDone.future
+
+        manifestFuture.onFailure {
+          case failure: Throwable =>
+            logWarn(s"error in batch $failure")
+            promiseZipDone.failure(failure)
+            batchFileState.onZipFileProcessed(zipFileName)
+        }
+        successfulJsonManifests map { manifestMessages =>
+          logInfo(s"got ${manifestMessages.length} manifest messages to send")
+
+          val manifestSource: Source[VoyageManifest, NotUsed] = Source(manifestMessages)
+          val zipSendingGraph: RunnableGraph[NotUsed] = manifestSource.to(subscriberFlightActor)
+          zipSendingGraph.run()
+
+          eventualZipCompletion.onFailure { case oc => log.error(logMsg(s"processingZip had error ${oc}")) }
+          eventualZipCompletion.onSuccess {
+            case zipFilename =>
+              logInfo(s"updating latestFile: from $latestFile to $zipFilename")
+              batchFileState.onZipFileProcessed(zipFileName)
+              logInfo(s"finished processing. latestFile now: ${batchFileState.latestFile}")
+          }
+        }
+        eventualZipCompletion
+      })
+  }
+
+  // todo rather than this List[ZipUtils.UnzippedFileContent] we should be trying to deal in Sources, if we do
+  // that should enable us to reduce some of the Future state management
+  private def manifestsToAdvPaxReporter(logInfo: (String) => Unit, advPaxReporter: ActorRef, manifests: List[ZipUtils.UnzippedFileContent]): Seq[(Option[String], String, Try[VoyageManifest])]
```

Comment:
rename as this no longer sends manifests to the advPaxReporter

remove the advPaxReporter from signature